script.js

```diff
@@ -0,0 +1,71 @@
+if (localStorage.getItem('data') === null) {
+  localStorage.setItem('data', '[]');
+}
+
+function Book(id, title, author) {
+  this.id = id;
+  this.title = title;
+  this.author = author;
+}
+
+function addBooks(bookList) {
+  const books = document.getElementById('books');
+  const liToRemove = document.querySelectorAll('li');
+  liToRemove.forEach((item) => {
+    item.remove();
+  });
+
```

Comment:
* The function addBooks is not adding books to a collection of books eg( An array) 
* i suggest you make this function add a book to a collection and not add books directly to the DOM